examples/cms-payload/app/(site)/[slug]/page.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
```

Comment:
This import seems to be unused. You don't need to import `React` anymore if you don't reference it, let's drop it here, and all other places as well. :+1: